_posts/2023-02-17-7th-workshop-announcement.md

```diff
@@ -0,0 +1,91 @@
+subtitle: 28 August - 1 September, Edinburgh, Scotland
+author: Beatriz Recinos, Anouk Vlug, Dan Goldberg, Fabien Maussion and Ben Marzeion
+date: 2023-02-17T00:00:00
+tags: workshop
+---
+
+### What is it ?
+
+**The workshop is the 7<sup>th</sup> in a series that has brought together the community of users 
+and developers of the Open Global Glacier Model ([OGGM](oggm.org)). This year's workshop 
+is organized by colleagues based at [The University of Edinburgh](https://www.ed.ac.uk/geosciences). It is open to anybody interested in global glacier modeling with focus on active developers and users of OGGM. The primary goal is to discuss the challenges 
+and solutions in modeling glaciers at large scales with OGGM, to learn from each other, and to develop 
+ideas and visions for the advancement of the model. The workshop will include scientific 
+presentations, hands-on coding sessions, discussion/brainstorming sessions, and generally 
+be held in an informal manner (with a possible hike to a Scotish National park).**
+
+### Agenda
+
+The workshop will include:
+
+1. <b>Scientific presentations</b> by the participants (ca. 15-20 min) on topics related to the theme of the workshop. 
+These can include the state, recent developments and results of glacier modeling with OGGM.
+2. <b>Discussion/brainstorming sessions</b> focused on topics of interest, such as future developments and visions for the model.
+3. <b>Hands-on sessions</b> in smaller groups including:
+   - Beginner tutorials: these sessions are for current and future OGGM users to learn how to set-up and customize a model run.
+   - Advanced tutorials: on topics such as <i>How to contribute to OGGM</i>, <i>use OGGM with your own data</i>, or <i>use parts of OGGM with another model</i>.
+   - Q&A session: on model problems and possible solutions.
+
+4.  <b>Workshop excursion</b>: Ice sheets covered all of Scotland in the past, leaving an amazing landscape behind. We plan to have a one-day excursion to one of Scotland's iconic ice carved landscapes.
+
+We would have the beginner hands-on sessions 
+at the first 1.5 days of the workshop, so new users of OGGM can get up to speed and have a better understanding 
+of what is coming next. For the rest of the week, (tentatively) we plan to have presentations mostly in the mornings and group discussions and hand-on sessions in the afternoons. 
+
+>>**Note for participants, GLAMBIE Workshop 28.08.2023 – 30.08.2023**:
+
+>> The second Glacier Mass Balance Intercomparison Exercise (GLAMBIE) workshop will take place also in Edinburgh during the first half of the week. This is a rare opportunity to connect the modelling and remote sensing communities! We will organize a joint session and social event either on Tuesday or Wednesday. GLAMBIE participants are welcome to register to the OGGM workshop if they are interested (we will do our best to accommodate all wishes, depending on room availability, for more information on accomodation see below).
```

Comment:
Can you add a link to GLAMBIE here? You may need to use HTML instead of markdown

Reply:
I did not know that there was a GLAMBIE website and is very fancy 😲 I shall do that now